src/NServiceBus.Core.Tests/Msmq/MsmqTestsBase.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace NServiceBus.Core.Tests.Msmq
+{
+    using System.Messaging;
+
+    public class MsmqTestsBase
```

Comment:
I don't see a lot of value in this base class. To be frank I'm not a big fan of base classes. When we keep it we should make it a bit more useful and potentially rename it `SingleMsmqQueueTestBase` and add the setup and the teardown of the queue to the base. I.ex. inheritors could override an abstract property called `QueueName` or something